tests/metadata-ext.rs

```diff
@@ -83,3 +83,55 @@ fn test_metadata_ext() {
         );
     }
 }
+
+#[test]
+fn test_metadata_ext_created() {
+    let tmpdir = tmpdir();
+    let a = check!(tmpdir.create("a"));
+    let a_metadata = check!(a.metadata());
+
+    let modified = check!(a_metadata.modified());
+    let tolerance = std::time::Duration::from_secs(10);
+    let expected = (modified - tolerance)..(modified + tolerance);
+
+    // If the standard library supports file creation times, then cap-std
+    // should too.
+    let std_supports_created = matches!(
+        a.into_std().metadata(),
+        Ok(m) if m.created().is_ok(),
+    );
```

Comment:
Instead of using the `modified()` timestamp for `expected`, could you use the `created()` timestamp for `expected`? And for the directories, could we always call `std::fs::metadata` so that we always get the `created()` timestamp from std? It seems like that should eliminate the need for a tolerance, because the creation time from cap-std should be exactly the same as the creation time from std.

Reply:
Yep, that makes more sense than what I had done. Updated.